kpm/lib/kpm/installer.rb

```diff
@@ -27,13 +34,12 @@ def self.build_default_config(all_kb_versions=nil)
         next if version.nil?
 
         major, minor, patch, pre = version.segments
-        next if !pre.nil? || minor.nil? || minor.to_i.odd?
+        next if !pre.nil? || minor.nil? || minor.to_i.odd? || patch.to_i.odd?
```

Comment:
No need to look at the patch version (0.18.11 is stable for instance -- see description at http://killbill.io/downloads/).